operators/multiclusterobservability/pkg/certificates/certificates.go

```diff
@@ -194,6 +194,7 @@ func createCACertificate(cn string, caKey *rsa.PrivateKey) ([]byte, []byte, erro
 	return caKeyBytes, caBytes, nil
 }
 
+//nolint:unparam
```

Comment:
Are there really unused params here?

Reply:
Yes, this one is valid `ou` is unused, but didn't want to refactor in this PR, as at first glance would need some logical changes. Added TODO.